File: Javascript Fundamentals/02.Conditional-Statements/QuadraticEquation.js
function solve(args) {
    let a = Number(args[0]);
    let b = Number(args[1]);
    let c = Number(args[2]);
    let discriminant = Math.pow(b, 2) - (4 * a * c);
    let x1, x2;


    if (discriminant < 0) {
        console.log("no real roots");
    }

    if (discriminant === 0) {
        x1 = (-b) / (2 * a);
        console.log("x1=x2=" + x1.toFixed(2));
    }

    else if (discriminant > 0) {
        x2 = ((-b) + Math.sqrt(discriminant)) / (2 * a);
        x1 = ((-b) - Math.sqrt(discriminant)) / (2 * a);
        console.log("x1=" + x1.toFixed(2) + "; x2=" + x2.toFixed(2));

    }
}
solve(['5', '2', '8']);

File: C# I/04.Console-In-and-Out/05.QuadraticEquation/QuadraticEquation.cs
using System;
    class QuadraticEquation
    {
        static void Main()
        {
            double a = double.Parse(Console.ReadLine());
            double b = double.Parse(Console.ReadLine());
            double c = double.Parse(Console.ReadLine());
            double discriminant = Math.Pow(b, 2) - 4 * a * c;
            double firstRoot;
            double secondRoot;

            if (discriminant < 0)
            {
                Console.WriteLine("no real roots");
            }

            else if (discriminant == 0)
            {
                firstRoot = -b / (2 * a);
                Console.WriteLine("{0:F2}", firstRoot);
            }

            else
            {
                firstRoot = (-b - Math.Sqrt(discriminant)) / (2 * a);
                secondRoot = (-b + Math.Sqrt(discriminant)) / (2 * a);
                Console.WriteLine("{0:F2}", firstRoot);
                Console.WriteLine("{0:F2}", secondRoot);
            }
        }
    }
